pennylane/operation.py

```diff
@@ -267,6 +267,7 @@
 
 SUPPORTED_INTERFACES = {"numpy", "scipy", "autograd", "torch", "tensorflow", "jax"}
 __use_new_opmath = False
+_UNSET_BATCH_SIZE = -1  # indicates that the (lazy) batch size has not yet been accessed/computed
```

Comment:
It's not _really_ necessary, but maybe the following could be safer, as it can't be obtained from a batch size calculation by accident?
```suggestion
_UNSET_BATCH_SIZE = object()  # indicates that the (lazy) batch size has not yet been accessed/computed
```

Reply:
never seen `object()` but I like it 🙌

Reply:
had to put it back, if you `deepcopy` the tape it makes a new `object` for the new tape's (unset) `_batch_size`

Reply:
haha, alright :face_with_spiral_eyes:

Reply:
I was disappointed but I don't want any trouble 😅